suponoff-monhelper.py

```diff
@@ -42,10 +42,15 @@ def getProcessResourceUsage(self, pid_list):
         try:
             results = {}
             for pid in pid_list:
+                if pid == 0:
+                    continue
                 result = {}
                 results[str(pid)] = result
                 try:
                     proc = psutil.Process(pid)
+                except psutil.NoSuchProcess as e:
+                    LOG.info("Process %s: %s", pid, e.msg)
```

Comment:
I am happy either way. Please notice that e.msg already describes the error, "Process %s: %s" gives: "Process 17487: no process found with pid 17487".